JPS_Ontology/ontology/ontodevice/OntoDevice.csv

```diff
@@ -153,11 +154,13 @@ hasSurvivalRange,Object Property,,,Device,http://www.w3.org/ns/ssn/systems/Survi
 hasQuantity,Object Property,,,ScreenBrightness,RelativeBrightness,only,"A relationship between something (it could be a device or space, a setpoint, an operating property etc) and the quantity it represents or has.",https://www.theworldavatar.com/kg/ontodevice/,has quantity
 hasTemperature,Object Property,hasQuantity,IS-A,https://w3id.org/bot#Zone UNION http://www.theworldavatar.com/ontology/ontocape/material/phase_system/phase_system.owl#SinglePhase,http://www.ontology-of-units-of-measure.org/resource/om-2/Temperature,,"A relationship describing the degree or intensity of heat present in a substance, object or space.",https://www.theworldavatar.com/kg/ontodevice/,has temperature
 hasRelativeHumidity,Object Property,hasQuantity,IS-A,https://w3id.org/bot#Zone,http://www.ontology-of-units-of-measure.org/resource/om-2/RelativeHumidity,,"A relationship describing the relative humidity of a substance, object or space.",https://www.theworldavatar.com/kg/ontodevice/,has relative humidity
+hasAcidity,Object Property,hasQuantity,IS-A,http://www.theworldavatar.com/kg/ontochemplant#Fluid,http://www.ontology-of-units-of-measure.org/resource/om-2/Acidity,,A relationship describing the acidity of a solution.,https://www.theworldavatar.com/kg/ontodevice/,has acidity
 hasPressure,Object Property,hasQuantity,IS-A,https://w3id.org/bot#Zone UNION http://www.theworldavatar.com/ontology/ontocape/material/phase_system/phase_system.owl#SinglePhase,http://www.ontology-of-units-of-measure.org/resource/om-2/Pressure,,"A relationship describing the pressure exerted on a substance, object or a space.",https://www.theworldavatar.com/kg/ontodevice/,has pressure
+hasAirExchangeRate,Object Property,https://www.theworldavatar.com/kg/ontodevice/hasQuantity,IS-A,https://saref.etsi.org/core/HVAC,http://www.theworldavatar.com/ontology/ontodevice/AIrChangePerHour,,A relationship that describes how many times does a volume of air equal to the volume of a room can an HVAC unit pump out of the room per hour,https://www.theworldavatar.com/kg/ontobms/,has air exchange rate
```

Comment:
is it ontodevice:AirChangePerHour or ontobms:AirChangePerHour? 

ontodevice:hasAirExchangeRate should be defined under ontodevice right?

Reply:
Another comment: to me, it will make more sense to say that a room or zone has air exchange rate, rather than a hvac device having air exchange rate.